### PR TITLE
Added CVE-2017-1000159

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

**Comment by cve-team** [comment]
Thank you for your submission. It does not appear that the description contains a version number, could you please add one?

**Comment by kurtseifried** [comment]
updated description

File: 2017/1000xxx/CVE-2017-1000159.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "CVE_data_meta": {
+        "ASSIGNER": "kurt@seifried.org",
+        "DATE_ASSIGNED": "2017-08-22T17:29:33.371820",
+        "ID": "CVE-2017-1000159",
+	"STATE": "REJECT",
```
Comment (cve-team) [inline 2017/1000xxx/CVE-2017-1000159.json]: Should STATE be "PUBLIC" now?

**Comment by cve-team** [comment]
Thanks. Look for the entry to be published on https://cve.mitre.org within the next few hours.

George

**Pull request merged**